mypy/semanal.py

```diff
@@ -468,7 +468,7 @@ def find_type_variables_in_type(self, type: Type) -> List[Tuple[str, TypeVarExpr
             for arg in type.args:
                 result.extend(self.find_type_variables_in_type(arg))
         elif isinstance(type, TypeList):
-            for item in type.items:
+            for item in type.types:
```

Comment:
Land #3113 first, then handle this conflict. Then see if we can remove the `types` property.